packages/crayons-core/src/components/country-phone/readme.md

```diff
@@ -0,0 +1,319 @@
+# FwCountryPhone(fw-country-phone)
```

Comment:
Rename this to just Country Phone. This will appear in documentation menu.

Reply:
Fixed ✅